cognite/seismic/protos/types.proto

```diff
@@ -23,6 +23,8 @@ message Trace {
     google.protobuf.Int32Value iline = 2;
     google.protobuf.Int32Value xline = 3;
     repeated float trace = 4;                // The underlying array of floats representing samples
+    float x = 5;
```

Comment:
The other things we have with x and y generally specify a crs as well. Could be a "Coordinate" type, containing x, y and crs. But that might be preoptimization and I would hate myself for that.

Reply:
done